README.rst

```diff
@@ -88,24 +88,24 @@ requirements need to be met:
 
 On Ubuntu systems, you can install these dependencies with::
```

Comment:
```suggestion
On Ubuntu 18.04 onwards, you can install these dependencies with::
```

Debain stable and "old" Ubuntu systems do not have the `gstreamer1.0-gtk3` package.